src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from 'next/link'
+
+const ProductCard = () => {
+  return (
+    <div className="product-card">
+      <Link href="/product-view">
```

Comment:
Since we are going to pass some props to each product item. I would consider using Next Router instead.